Library/Formula/flatcc.rb

```diff
@@ -0,0 +1,54 @@
+class Flatcc < Formula
+  desc "FlatBuffers Compiler and Library in C for C"
+  homepage "https://github.com/dvidelabs/flatcc"
+  url "https://github.com/dvidelabs/flatcc/archive/v0.2.0.tar.gz"
+  sha256 "5327e0c0faa37643f59be6f086576ef76a13d36b655d2bf9547f89ffce3b3413"
+  head "https://github.com/dvidelabs/flatcc.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "-G", "Unix Makefiles", buildpath, *std_cmake_args
+    system "make"
+
+    bin.install "bin/flatcc"
```

Comment:
Author of flatcc here. I much appreciate your on providing support for this. It is not likely that flatcc will be integrated into the main flatbuffers project due to the challenges listed here https://github.com/dvidelabs/flatcc/issues/1

The README does provide clear install instructions on ninja, but the project is still evolving. Post 0.2.0 (no release yet) it was made simpler to use make instead of ninja because a user had issues with this on Centos. https://github.com/dvidelabs/flatcc#building

The build script is primarily there to avoid users having to learn about cmake, and to extract build products. You can run CMake builds directly, but then you have to extract the build products similar to how the build script does it - the bin and lib folders are then not created automatically.

I'd be happy to accept a makefile for the specific purpose of supporting package managers which I have no experience in dealing with, if you think it is actually beneficial over running a shell script. I'd imagine it will act as a driver to the build scripts and use make as the build tool.